Assets/Animations.meta

```diff
@@ -1,9 +0,0 @@
-fileFormatVersion: 2
```

Comment:
Hmm weird. We can look into that. You could use git checkout origin/master -- file path to check them out and push the originals. This will exclude them from this PR until we fix it